<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DS8000 Programming</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="configBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuration</string>
    </property>
    <widget class="QPushButton" name="setIPAdaptButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Set IPv4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetNetAdaptButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset IPv4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="skipNetButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Skip Config</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>421</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="consoleOutput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>421</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="excelBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Excel Sheet</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="newExcelButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Create New</string>
     </property>
    </widget>
    <widget class="QPushButton" name="uploadExcelButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>enableConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setIPAdaptButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetNetAdaptButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>skipNetButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>skipConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newExcelButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>newExcel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadExcelButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openUpload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableConfig()</slot>
  <slot>startConfig()</slot>
  <slot>resetConfig()</slot>
  <slot>fillOutput()</slot>
  <slot>skipConfig()</slot>
  <slot>newExcel()</slot>
  <slot>openUpload()</slot>
 </slots>
</ui>
